components/builder-api/src/server/error.rs

```diff
@@ -176,6 +176,7 @@ impl ResponseError for Error {
             Error::OAuth(_) => HttpResponse::new(StatusCode::UNAUTHORIZED),
             Error::ParseIntError(_) => HttpResponse::new(StatusCode::BAD_REQUEST),
             Error::Protocol(_) => HttpResponse::new(StatusCode::INTERNAL_SERVER_ERROR),
+            Error::DieselError(ref e) => HttpResponse::new(diesel_err_to_http(&e)),
```

Comment:
ooh you fancy huh? ![](https://media.giphy.com/media/5xtDarEB54QzKau1InC/giphy-downsized.gif)